src/main/java/de/uni_tuebingen/ub/nppm/servlet/RESTServlet.java

```diff
@@ -0,0 +1,94 @@
+                response.getWriter().write("Please specify name parameter");
+            }
+            else if(entity != null && entity.compareTo("mghlemma") == 0) {
+                List<MghLemma> lemmas = null;                
+                try {
+                    lemmas = MghLemmaDB.getByName(name);
+                } catch (Exception ex) {
+                    response.getWriter().write(ex.getLocalizedMessage());                    
+                }
+
+                if (lemmas.size() > 0) {
+                    try {
+                        String json = "\n{\n";
+                        json += " [\n";
+                        for(MghLemma lemma: lemmas){
+                            json += "  {\n";
+                            json += "  \"mghLemma\": " + JSONObject.quote(lemma.getMghLemma()) + ",\n";
+                            json += "  \"id\": " + JSONObject.quote(String.valueOf(lemma.getId())) + ",\n";                            
+                            json += "  },\n";                            
+                        }
+                        json = json.substring(0, json.length() - 1);
+                        json += " \n ]\n";
+                        json += "}\n";
+                        response.getWriter().println(json);
+                    } catch (Exception e) {
+                        response.getWriter().write(e.getLocalizedMessage());
+                    }
+                }
+            }else if(entity.compareTo("namenkommentar") == 0){
+                List<NamenKommentar> namenkommentare = null;                
+                try {
+                    namenkommentare = NamenKommentarDB.getByName(name);
+                } catch (Exception ex) {
+                    response.getWriter().write(ex.getLocalizedMessage());                    
+                }
+
+                if (namenkommentare.size() > 0) {
+                    try {
+                        String json = "\n{\n";
+                        json += " [\n";
+                        for(NamenKommentar nk: namenkommentare){
+                            json += "  {\n";
+                            json += "  \"ELemma\": " + JSONObject.quote(nk.geteLemma()) + ",\n";
+                            json += "  \"PLemma\": " + JSONObject.quote(nk.getpLemma()) + ",\n";
+                            json += "  \"id\": " + JSONObject.quote(String.valueOf(nk.getId())) + ",\n";                            
+                            json += "  },\n";                            
+                        }
+                        json = json.substring(0, json.length() - 1);
+                        json += "\n ]\n";
+                        json += "}\n";
+                        response.getWriter().println(json);
+                    } catch (Exception e) {
+                        response.getWriter().write(e.getLocalizedMessage());
+                    }
+                }
+            }
```

Comment:
We should definitely add an example for "Einzelbeleg", since this will be the most important case for the community.

Reply:
yes einzelbeleg is implemented now